cypress/integration/embedded-mode.spec.ts

```diff
@@ -1,20 +1,6 @@
-import { GetSourceFileSsrQuery } from '../../src/graphql/GetSourceFileSSR.generated';
-
-const SOURCE_ID = 'source-file-id';
-
-const getSSRParam = (
-  data: Partial<GetSourceFileSsrQuery['getSourceFile']> & { id: string },
-) => {
-  return encodeURIComponent(JSON.stringify({ data, id: data.id }));
-};
-
-describe('Embedded mode', () => {
-  describe('default (mode:viz)', () => {
-    before(() => {
-      cy.interceptGraphQL({
-        getSourceFile: {
-          id: SOURCE_ID,
-          text: `
+const sourceFileFixture = {
```

Comment:
no functional changes in this file - I've just created typed `cy.visitEmbedWithNextPageProps` and used it here (since I had to adjust this code anyway to cover for `GetSourceFileSsrQuery` being removed